gui/src/components/NewVersionNotice.tsx

```diff
@@ -49,16 +49,23 @@ export function NewVersionNotice() {
     setState({ ...state, open: false });
   };
 
-  if (!latest) return null;
+  if (!latest || current === latest) return null;
 
-  return current !== latest ? (
+  return (
     <Snackbar
       anchorOrigin={{ vertical, horizontal }}
       open={open}
       onClose={handleClose}
       key={vertical + horizontal}
     >
-      <Link href="https://github.com/iron-wallet/iron/releases" target="_blank">
+      <Link
+        href="https://github.com/iron-wallet/iron/releases"
+        sx={{
+          color: "inherit",
+          textDecorationColor: "currentColor !important",
```

Comment:
are we using `!important` already? 😕

Reply:
Unfortunately yes :/ I tried a lot of things. The `text-decoration-color` has a huge priority, and it's impossible to override without making a mess. This was the most harmless change I managed because I'm forcing the color of the underline to be whatever the current color is. I'm not even setting the color itself.

Reply:
apparently using `&&` worked https://dev.to/magischerzwerg/double-ampersand-trick-in-sass-with-react-4khe